packages/storefront-events-collector/src/handlers/search/searchRequestSentAEP.ts

```diff
@@ -61,6 +62,10 @@ const handler = async (event: Event): Promise<void> => {
         value: 1,
     };
 
+    if (searchInputCtx) {
+        payload.searchRequest.id = searchInputCtx.data.searchRequestId as string;
```

Comment:
why are we casting this as a string? is it not defined in the types anywhere?

Reply:
@grahamcrackers i did not chase it down, my change was moving this existing line from elsewhere